### PR TITLE
[ECHOJS-91] API connections tests

### PR DESCRIPTION
**Pull request by adziaryba**
API connection tests for the next cases:
-using non-connected api;
-default apis connections;
-using nonexistent api;
-reconnect valid apis;

### CODE REVIEW

File: test/api.test.js [hunk 1]
```diff
@@ -5,11 +5,112 @@ import Cache from '../src/echo/cache';
...
+				await echo.disconnect();
+			})
+		});
+
+		describe('using nonexistent api', () => {
+			it('should throw error', async () => {
+				try {
+					await echo.connect(url, {
+						apis: [
+							'artsiomka'
```
Comment (nicklatkovich) [inline test/api.test.js]: Change to "nonexistent"

**Review by nicklatkovich — CHANGES_REQUESTED** [review]
Change titles of "describe" and "it".
"should work" is not good title for test.
See first describe for example

File: test/api.test.js (hunk 1 above)
Comment (adziaryba) [reply test/api.test.js]: fixed, i guess so

**Comment by nicklatkovich** [comment]
@adziaryba , for the future. Press "re-request review" when you fix all the comments of the previous review.

**Review by nicklatkovich — APPROVED** [review]

**Pull request merged**